docs/book/releases/v2.1.0.md

```diff
@@ -30,10 +30,10 @@ Note: For vSphere CSI Migration feature the minimum Kubernetes version requireme
 
 1. When the static persistent volume with file-share is re-created with the same PV name, volume is not getting registered as a container volume with vSphere.
    - Impact: attach/delete can not be performed on the such Persistent Volume.
-   - Workaround: wait for 1 hour before re-creating static persistent volume with file-share using the same name.
+   - Workaround: wait for 1 hour before re-creating static persistent volume with file-share using the same name. Note: This issue is fixed in [v2.2.0](v2.2.0.md)
```

Comment:
Can the "workaround" be to upgrade the CSI driver to v2.2.0? And if thats not possible then workaround 2 would be to wait 1 hour

Reply:
since upgrade can not be a workaround, it is a proper fix. so added this as a note.